src/bmconfigparser.py

```diff
@@ -69,6 +69,7 @@ def set(self, section, option, value=None):
             raise ValueError("Invalid value %s" % value)
         return SafeConfigParser.set(self, section, option, value)
 
+    # pylint: disable=redefined-builtin, too-many-return-statements
     def get(self, section, option, raw=False, vars=None):
```

Comment:
This method shouldn't be written in the first place. It's an unneeded crutch. `redefined-builtin` is not good. Pylint shows `too-many-return-statements` warning because it's poorly written, through a brute force. The same applies to `arguments-differ`. Better to remove it completely and refactor the `BMConfigParser` in a proper way.